.env.demo

```
@@ -1,140 +1,136 @@
-KEYCLOAK_MASTER_REALM=master
-KEYCLOAK_MANAGEMENT_CLIENT_ID=adminClient
-KEYCLOAK_MANAGEMENT_CLIENT_SECRET=
-KEYCLOAK_REALM=credebl-platform
-
-SCHEMA_FILE_SERVER_URL=
-SCHEMA_FILE_SERVER_TOKEN=
-
-GEO_LOCATION_MASTER_DATA_IMPORT_SCRIPT=/prisma/scripts/geo_location_data_import.sh
-UPDATE_CLIENT_CREDENTIAL_SCRIPT=/prisma/scripts/update_client_credential_data.sh
+X509_NKEY_SEED= # Please provide Nkeys secret for x509 service
+OIDC4VC_ISSUANCE_NKEY_SEED= # Please provide Nkeys secret for x509 service
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)